packages/react-components/styles/styles.ts

```diff
@@ -2,7 +2,7 @@ import colors from '@celo/react-components/styles/colors'
 import { fontFamily } from '@celo/react-components/styles/fonts'
 import { StyleSheet } from 'react-native'
 
-export const TOP_BAR_HEIGHT = 45
+export const TOP_BAR_HEIGHT = 56
```

Comment:
Given that the only consumer of this is mobile, I feel this should be moved there out of react-components